src/js/components/conversation.jsx

```diff
@@ -136,13 +136,18 @@ export class ConversationComponent extends Component {
             }
         }
         this._lastTopMessageNode = undefined;
-        
+
     };
 
     componentWillUpdate(nextProps) {
         const {messages: currentMessages, isFetchingMoreMessages} = this.props;
         const {messages: newMessages} = nextProps;
 
+        if (!this._lastMessageNode) {
+            this._forceScrollToBottom = true;
```

Comment:
Why do we need to scroll to bottom when undefined? Why not leave the conversation view as-is?